Makefile

```diff
@@ -7,7 +7,7 @@ test:
 	python setup.py test
 
 lint:
-	flake8 vyper tests --ignore=E122,E124,E127,E128,E501,E731,W504
+	flake8 vyper tests --ignore=E122,E124,E127,E128,E201,E202,E203,E501,E731,W504
```

Comment:
I think E201 / E202 / E203 can be fixed, instead of ignored?

Reply:
Oh - I find the extra whitespace aesthetically pleasing and I didn't see any written rationale for the existing ignored flags so I figured it wouldn't hurt to add them to the ignored flags. I can take them back out though :smile: